subscriptions-api.md

```diff
@@ -387,6 +387,21 @@ Each subscription is represented by an object that has the following properties:
 - Examples:
   - `https://example.com/event-processor`
 
+##### sinkCredential
```

Comment:
Do we need to say how these credentials are manifested on the wire?

Reply:
No. Depends on the protocol and negotiations. Might be a SASL choice.